src/main/java/com/dev/theater/Main.java

```diff
@@ -55,11 +59,15 @@ public static void main(String[] args) throws AuthenticationException {
         authenticationService.register("test2@bigmir.net", "12345");
         System.out.println(authenticationService.login("test2@bigmir.net", "12345").toString());
         userService.getAll().forEach(System.out::println);
-        ShoppingCart shoppingCart = shoppingCartService.getByUser(user);
         shoppingCartService.addSession(movieSession, user);
+        Ticket ticket = new Ticket();
+        ticket.setUser(user);
+        ticket.setMovieSession(movieSession);
+        ShoppingCart shoppingCart = shoppingCartService.getByUser(user);
+        shoppingCart.getTicketList().add(ticket);
```

Comment:
let's remove this and use `addSession()` instead

Reply:
done